core/Cargo.toml

```diff
@@ -381,6 +381,20 @@ prometheus-client = { version = "0.22.2", optional = true }
 tracing = { version = "0.1", optional = true }
 # for layers-dtrace
 probe = { version = "0.5.1", optional = true }
+wasm-bindgen = "0.2.95"
```

Comment:
Hi, please mark those dependencies as `optional` and hide them under the `services-opfs` feature to ensure other builds aren't affected.